packages/web/src/components/track/mobile/ConnectedPlaylistTile.tsx

```diff
@@ -167,14 +166,6 @@ const ConnectedPlaylistTile = ({
     [goToRoute, getRoute]
   )
 
-  const goToArtistPage = useCallback(
-    (e: MouseEvent<HTMLElement>) => {
-      e.stopPropagation()
```

Comment:
ok to lose stop propagating?

Reply:
yes cause the Link component stops prop by default, confirmed clicking the links didnt start playing a track